specification/quantum/data-plane/Microsoft.Quantum/preview/2021-05-06-preview/quantum.json

```diff
@@ -0,0 +1,757 @@
+          "description": "The target identifier to run the job."
+        },
+        "metadata": {
+          "type": "object",
+          "additionalProperties": {
+            "type": "string"
+          },
+          "description": "The job metadata. Metadata provides client the ability to store client-specific information"
+        },
+        "outputDataUri": {
+          "type": "string",
+          "description": "The output blob SAS uri. When a job finishes successfully, results will be uploaded to this blob."
+        },
+        "outputDataFormat": {
+          "type": "string",
+          "description": "The format of the output data."
+        },
+        "status": {
+          "type": "string",
+          "description": "The job status.",
+          "readOnly": true,
+          "enum": [
+            "Waiting",
+            "Executing",
+            "Succeeded",
+            "Failed",
+            "Cancelled"
+          ],
+          "x-ms-enum": {
+            "name": "JobStatus",
+            "modelAsString": true
+          }
+        },
+        "creationTime": {
+          "type": "string",
+          "readOnly": true,
+          "format": "date-time",
+          "description": "The creation time of the job."
+        },
+        "beginExecutionTime": {
+          "type": "string",
+          "readOnly": true,
+          "format": "date-time",
+          "description": "The time when the job began execution."
+        },
+        "endExecutionTime": {
+          "type": "string",
+          "readOnly": true,
+          "format": "date-time",
+          "description": "The time when the job finished execution."
+        },
+        "cancellationTime": {
+          "type": "string",
+          "readOnly": true,
+          "format": "date-time",
+          "description": "The time when a job was successfully cancelled."
+        },
+        "errorData": {
+          "$ref": "#/definitions/ErrorData",
+          "readOnly": true,
+          "description": "The error data for the job. This is expected only when Status 'Failed'."
+        },
+        "tags": {
+          "type": "array",
+          "items": {
+            "type": "string"
+          },
+          "description": "List of user-supplied tags associated with the job."
+        }
+      },
+      "required": [
+        "containerUri",
+        "inputDataFormat",
+        "providerId",
+        "target"
+      ]
+    },
+    "BlobDetails": {
+      "description": "Blob details.",
+      "type": "object",
+      "properties": {
+        "containerName": {
+          "type": "string",
+          "description": "The container name."
+        },
+        "blobName": {
+          "type": "string",
+          "description": "The blob name."
+        }
+      },
+      "required": [
+        "containerName"
+      ]
+    },
+    "SasUriResponse": {
+      "description": "Get SAS URL operation response.",
+      "type": "object",
+      "properties": {
+        "sasUri": {
```

Comment:
If this is a response only model, should we mark uri as "required"

Reply:
Out of scope for this PR since I didn't modify this area.